test/unit/lodash.spec.js

```diff
@@ -0,0 +1,583 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+
+describe('Lodash training', function() {
+    var params;
+
+    describe('functions', function() {
+        beforeEach(function() {
+            params = datasets.functions();
+        });
+        it('should match types of passing elements', function() {
+            var obj = params[0];
+            expect(obj).to.be.an('object');
+            expect(params).to.have.length(1);
+        });
+        it('should contain only one property square', function() {
+            var obj = params[0];
+            var i = 0;
+            for (var key in obj) {
+                if (obj.hasOwnProperty(key) && key === 'square') {
+                    if (obj[key] instanceof Function) {
+                        i++;
+                    }
+                }
+            }
+            expect(i).to.eql(1);
+        });
+        it('should not contain own property cube', function() {
+            var obj = params[1];
+            for (var key in obj) {
+                if (obj.hasOwnProperty(key)) {
+                    expect(key).to.not.eql('cube');
+                }
+            }
+        });
+        it('should return array property names of functions', function() {
+            expect(_.functions.apply(_, params)).to.eql(['square']);
+        });
+    });
+
+    describe('functionsIn', function() {
+        beforeEach(function() {
+            params = datasets.functionsIn();
+        });
+        it('should match types of passing elements', function() {
+            var element = params[0];
+            expect(element).to.be.an('object');
+            expect(params).to.have.length(1);
+        });
+        it('should contain only one property square', function() {
```

Comment:
This case is incorrect because of 2 things:
- first of all, this is basically checked by functionsIn lodash method so what you do here is just duplicate the check
- name of this case should be assigned to the actual check with lodash execution instead of how it looks right now